src/WebJobs.Script/Workers/Http/Configuration/HttpWorkerOptions.cs

```diff
@@ -29,5 +32,19 @@ public class HttpWorkerOptions
         public bool EnableProxyingHttpRequest { get; set; }
 
         public TimeSpan InitializationTimeout { get; set; } = TimeSpan.FromSeconds(30);
+
+        public string Format()
```

Comment:
Similar to [HostHealthMonitorOptions](https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Config/HostHealthMonitorOptions.cs).